.mci.yml

```diff
@@ -752,6 +775,32 @@ tasks:
         - func: "run_mongohouse"
         - func: "test_mongohouse"
 
+    - name: test_versioned_api
+      commands:
+        - func: "setup"
+        - func: "start_mongod"
+          vars:
+              REQUIRE_API_VERSION: true
+              # Authentication with versioned API should already be tested
+              # in the C driver.
+              AUTH: noauth
```

Comment:
While we _should_ be testing with authentication (see versioned API test README [here](https://github.com/mongodb/specifications/blob/master/source/versioned-api/tests/README.rst#testing-with-required-api-version)), we would have to add username and password to every URI used in testing. Versioned API on authentication-related commands is handled entirely through the C driver, so as long as we test with authentication in the C driver, I don't think we need to do so here.